samtranslator/model/stepfunctions/events.py

```diff
@@ -97,16 +97,15 @@ def to_cloudformation(self, resource, **kwargs):
 
         permissions_boundary = kwargs.get("permissions_boundary")
 
-        events_rule = EventsRule(self.logical_id)
+        passthrough_resource_attributes = resource.get_passthrough_resource_attributes()
+        events_rule = EventsRule(self.logical_id, attributes=passthrough_resource_attributes)
         resources.append(events_rule)
 
         events_rule.ScheduleExpression = self.Schedule
         if self.Enabled is not None:
             events_rule.State = "ENABLED" if self.Enabled else "DISABLED"
         events_rule.Name = self.Name
         events_rule.Description = self.Description
-        if CONDITION in resource.resource_attributes:
-            events_rule.set_resource_attribute(CONDITION, resource.resource_attributes[CONDITION])
```

Comment:
These lines removed because attributes are sent to `events_rule` object at line 101, right?

Reply:
right